### PR TITLE
feat: remove experimental flag for telemetry

### PR DESCRIPTION
**Pull request by daveleek**
## About the changes

This removes the experimental feature flag that defaulted to turn off telemetry collection

### CODE REVIEW

File: src/lib/__snapshots__/create-config.test.ts.snap [hunk 1]
```diff
@@ -95,7 +94,7 @@ exports[`should create default config 1`] = `
       "proPlanAutoCharge": false,
       "responseTimeWithAppNameKillSwitch": false,
       "segmentContextFieldUsage": false,
-      "strategySplittedButton": false,
+      "strategySplittedButton": true,
```
Comment (sighphyre) [inline src/lib/__snapshots__/create-config.test.ts.snap:97]: What's going on here? Doesn't look related?

Comment (daveleek) [reply src/lib/__snapshots__/create-config.test.ts.snap:97]: Thank you, shouldn't have made it into this PR

**Review by sighphyre — APPROVED** [review]
LGTM
